src/aio/aio_theme_changer.py

```diff
@@ -34,16 +35,27 @@ class ids:
             "subcomponent": "radio",
             "aio_id": aio_id,
         }
-        dummy_div = lambda aio_id: {
+        store = lambda aio_id: {
             "component": "ThemeChangerAIO",
-            "subcomponent": "dummy_div",
+            "subcomponent": "store",
+            "aio_id": aio_id,
+        }
+        assetsPath = lambda aio_id: {
+            "component": "ThemeSwitchAIO",
+            "subcomponent": "assetsPath",
             "aio_id": aio_id,
         }
 
     ids = ids
 
     def __init__(
-        self, radio_props={}, button_props={}, offcanvas_props={}, aio_id=None,
+            self,
+            aio_id: str = str(uuid.uuid4()),
+            custom_themes: dict[str, str] = None,
```

Comment:
This requires at least python 3.9.  We should still support 3.8 until Dash drops that support.  Probably need to update the ThemeSwitch component too.

Reply:
Last commit adds backward compatibility with python 3.8.0